tests/tfm/tfm_psa_test/sample.yaml

```diff
@@ -0,0 +1,26 @@
+common:
+    tags: tfm
+    platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf9160dk_nrf9160_ns
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "\\*\\*\\*\\*\\* PSA Architecture Test Suite - Version .* \\*\\*\\*\\*\\*"
+        - "TOTAL FAILED *: 0"
+sample:
+  name: "TFM PSA Test"
+
+tests:
```

Comment:
The current "path" is `tests/sample/tfm/...` for the below entries.

Could you change both `sample.yaml` files to be called `testcase.yaml`, and remove the `sample` prefix from the entries?